src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -28,15 +31,16 @@
    * ".rscc" is a hidden folder in the user's home directory (e.g. /home/user)
    */
   private static final String RSCC_FOLDER_NAME = ".rscc";
+  private static final String STUN_DUMP_FILE_NAME = "ice4jDemoDumpe.ice";
```

Comment:
I think this is not needed, regarding the next TODO. Only the path to the file is used later on, an needs a proper implementation. This could be forgotten during refactoring, maybe you could add a comment.